examples/benchmark/script.jl

```diff
@@ -0,0 +1,43 @@
+using OptimalTransport
+using PythonOT: PythonOT
+using LinearAlgebra
+using Random
+using Tulip, Clp
+using BenchmarkTools
+
+Random.seed!(0)
+
+N = 100
+μ = fill(1 / N, N)
+ν = fill(1 / N, N)
+C = rand(N, N)
+
+# Exact 
+@benchmark begin
+    emd2(μ, ν, C, Tulip.Optimizer())
+end
```

Comment:
For accurate benchmarks one should interpolate global variables. There is some explanation and example in the README: https://github.com/JuliaCI/BenchmarkTools.jl#quick-start

Reply:
Apologies, this script shouldn't even be there (I thought it was in a different local branch!). Removing in next commit, but thanks for the link!